asset-transfer-basic/chaincode-external/README.md

```diff
@@ -159,16 +159,14 @@ Now that we have started the chaincode service and deployed it to the channel, w
 
 ## Using the Asset-Transfer-Basic external chaincode
 
-Open yet another terminal and navigate to the `fabric-samples/asset-transfer-basic/application-javascript` directory:
+Open yet another terminal and navigate to the `fabric-samples/asset-transfer-basic/application-gateway-go` directory:
 ```
-cd fabric-samples/asset-transfer-basic/application-javascript
+cd fabric-samples/asset-transfer-basic/application-gateway-go
```

Comment:
Any reason to use application-gateway-go instead of application-gateway-typescript? It seems more new users are familiar with javascript/typescript which is probably why the old instructions used javascript.

Reply:
Any of the client application languages implementations will work fine. I figured that, since this chaincode sample uses Go, the Go client application probably made more sense. One language rather than two.